### ISSUE TITLE
Numpy 2.0 compatibility

### ISSUE THREAD

**Issue by esalx** [SEDenmarkLab/molli#137]
I am not sure if the most recent numpy major update breaks molli in any way, but I think it would be best to test it and figure out a way to make the interfaces compatible. @blakeo2 thoughts?

**Comment by blakeo2** [comment]
You're totally right. With the changes made, I've run into error messages in the CDXML parser; however, this was not detected by Molli, this was instead detected due to the scipy implementation. I don't know how simple it's going to be introduce stress tests if they are necessarily a result of separate package incompatibility. At best we can take a look at things native to Molli and that might be the best next step

**Comment by esalx** [comment]
Could you share the error that you got from SciPy? We can also modify the dependency list to exclude numpy 2.0 until we can get a definitive support from all key dependencies of molli.

**Comment by blakeo2** [comment]
This issue appears to be resolved, as numpy 2.0 no longer conflicts with any key compatibilities of molli
